### PR TITLE
Apply fix_push_structure_to_global_registry to v0.0.2 -> v0.0.2.1

### PR DESCRIPTION
**Pull request by knutsenm**

### CODE REVIEW

**Comment by knutsenm** [comment]
This applies my patch to the version of cru_lib that infobase uses.

Upgrading infobase to the newer cru_lib would require rewriting its use of active_model_serializers to conform to v0.10.0rc

File: lib/cru_lib/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module CruLib
-  VERSION = "0.0.2"
+  VERSION = "0.0.2.1"
```
Comment (twinge) [inline lib/cru_lib/version.rb]: Why not go to 0.0.3?

Comment (knutsenm) [inline lib/cru_lib/version.rb]: 0.0.3 introduced active_model_serializers
> Upgrading infobase to the newer cru_lib would require rewriting its use of active_model_serializers to conform to v0.10.0rc

**Comment by twinge** [comment]
:+1:

**Comment by knutsenm** [comment]
OK, I tagged v0.0.2.1, please update rubygems

**Pull request closed without merging**